crates/context_aware_config/src/api/context/handlers.rs

```diff
@@ -633,15 +633,15 @@ async fn list_contexts(
     }
 
     let dimension_params = dimension_params.into_inner();
-
-    let mut builder = contexts.into_boxed();
-    match filter_params.sort_by.unwrap_or_default() {
-        ContextFilterSortBy::PriorityAsc => builder = builder.order(priority.asc()),
-        ContextFilterSortBy::PriorityDesc => builder = builder.order(priority.desc()),
-        ContextFilterSortBy::CreatedAtAsc => builder = builder.order(created_at.asc()),
-        ContextFilterSortBy::CreatedAtDesc => builder = builder.order(created_at.desc()),
-    }
-
+    let builder = contexts.into_boxed();
+
+    #[rustfmt::skip]
+    let mut builder = match (filter_params.sort_on.unwrap_or_default(), filter_params.sort_by.unwrap_or(SortBy::Asc)) {
```

Comment:
```suggestion
    let mut builder = match (filter_params.sort_on.unwrap_or_default(), filter_params.sort_by.unwrap_or_default) {
```
We already have a default for this to `Desc`, can use it for consistency.
Now if we want Asc to be default, better to change there first.